src/main/java/com/zerobase/ecommerce/application/SignUpApplication.java

```diff
@@ -0,0 +1,82 @@
+package com.zerobase.ecommerce.application;
+
+import com.zerobase.ecommerce.client.MailgunClient;
+import com.zerobase.ecommerce.client.mailgun.SendMailForm;
+import com.zerobase.ecommerce.domain.SignUpForm;
+import com.zerobase.ecommerce.domain.model.Customer;
+import com.zerobase.ecommerce.domain.model.Seller;
+import com.zerobase.ecommerce.exceiption.CustomException;
+import com.zerobase.ecommerce.exceiption.ErrorCode;
+import com.zerobase.ecommerce.service.customer.SignUpCustomerService;
+import com.zerobase.ecommerce.service.seller.SellerService;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class SignUpApplication {
+    private final MailgunClient mailgunClient;
+    private final SignUpCustomerService signUpCustomerService;
+    private final SellerService sellerService;
+
+    public void customerVerify(String email, String code){
+        signUpCustomerService.verifyEmail(email, code);
+    }
+
+    public String customerSignUp(SignUpForm form){
+        if(signUpCustomerService.isEmailExist((form.getEmail()))){
```

Comment:
```suggestion
        if (signUpCustomerService.isEmailExist(form.getEmail())) {
```
coding convention 을 지켜보면 어떨까요? 소괄호가 조금 많이 들어간 것 같은데 수정해주시면 좋겠습니다